### PR TITLE
13487 Size the entity table according to window height

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#13487

*Description of changes:*
- app version = 1.1.11
- changed a multi-line span to a div
- changed entity table height to CSS calculation to fit more height
- changed #fff to white

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Minimum table height example (short window)
image

Minimum table height example (tall window)
image

Example with few items
image

Example with many items (short window)
image

Example with manu items (tall window)
image

File: auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue [hunk 1]
```diff
@@ -110,8 +109,11 @@
         </template>
         <template v-slot:no-data>
           <span v-if="loading">Loading...</span>
-          <span v-else>Add an existing company, cooperative or society to manage it or<br> add a Name Request to
-            complete your incorporation or registration.</span>
+          <div v-else>
+            Add an existing company, cooperative or society to manage it or
+            <br>
+            add a Name Request to complete your incorporation or registration.
+          </div>
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:116]: There's really little difference here, but it's better to not pretend this is an inline element when it's actually a block-level element.

File: auth-web/src/components/pay/PayWithCreditCard.vue [hunk 2]
```diff
@@ -57,7 +57,7 @@ export default class PayWithCreditCard extends Vue {
     .payee-name {
       padding-right: 16px;
       margin-right: 16px;
-      border-right: 1px solid #fff;
```
Comment (severinbeauvais) [inline auth-web/src/components/pay/PayWithCreditCard.vue:60]: `white` is commonly used in this UI so I'm making this consistent.

**Comment by severinbeauvais** [comment]
I also tested this in FF. Looks great!

**Review by seeker25 — APPROVED** [review]
Looks great

File: auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue [hunk 3]
```diff
@@ -530,6 +528,17 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
...
+  // size the table according to window height
+  ::v-deep .v-data-table__wrapper {
+    min-height: 9rem; // height of no-data div
+    height: calc(100vh - 31rem); // full height minus footer
+    background-color: $gray1; // hide unused space below table
+
+    table {
+      background-color: white;
+    }
+  }
```
Comment (severinbeauvais) [inline auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:540]: @cameron-freshworks FYI, little trick for better space management :)

Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:540]: @severinbeauvais Very cool!

Comment (severinbeauvais) [reply auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:540]: Cause between you and me, we see a lot of tables 😆

Comment (cameron-freshworks) [reply auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:540]: We do indeed 😆

**Pull request merged**